### DISCUSSION TITLE
Disable Location Permission Request

### DISCUSSION THREAD

**Discussion by abumubaarak** [rnmapbox/maps#2605] [Q&A] [answered]
Hello everyone,

How can i disable location permission request on android and iOS?

**Comment by mfazekas** [comment, marked as answer]
Which version are you using? There was a bug that always asked for location permission, but it was fixed in the last beta.

**Reply by abumubaarak** [reply]
Thanks for the reply.

Unfortunately i noticed that i had maplibre-gl-native-distribution as part of my package dependency, the issue was solved after removing it.

**Comment by abumubaarak** [comment]
Thanks for the reply.

Unfortunately i noticed that i had maplibre-gl-native-distribution as part of my package dependency, the issue was solved after removing it.
